<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncomeWidget</class>
 <widget class="QWidget" name="IncomeWidget">
  <property name="windowTitle">
   <string>Доходы</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="editDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>2025-01-01</date>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSource">
       <property name="text">
        <string>Источник:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editSource"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAmount">
       <property name="text">
        <string>Сумма:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="editAmount">
       <property name="maximum">
        <double>1000000.00</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>Пользователь:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboUser"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnAddIncome">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableIncome">
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Источник</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сумма</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

